.github/workflows/build.yml

```
@@ -67,10 +59,76 @@ jobs:
             build/${{ matrix.version }}/report.json
             build/${{ matrix.version }}/progress-commit.json
 
+  verifyNonMatching:
```

```suggestion
  build-non-matching:
```
For consistency with `build` and avoiding camelCase